README.md

```diff
@@ -65,7 +66,7 @@ swift create-xcframework --platform ios --platform macos ...
 
 #### Catalyst
 
-You can build your XCFrameworks with support for Mac Catalyst by specifying `--platform maccatalyst` on the command line. As you can't include or exclude Catalyst support in your `Package.swift` we don't try to build it automatically.
+You can build your `XCFrameworks` with support for Mac Catalyst by specifying `--platform maccatalyst` on the command line. As you can't include or exclude Catalyst support in your `Package.swift` we don't try to build it automatically.
```

Comment:
XCFrameworks is a name of the product not a file name or code reference so I don't think the backticks make sense here, and they're inconsistent with the rest of the document.

Reply:
Got it, changing it

Reply:
Done